lib/matplotlib/rcsetup.py

```diff
@@ -775,6 +775,21 @@ def validate_hatch(s):
 validate_dashlist = _listify_validator(validate_floatlist)
 
 
+def validate_minor_tick_ndivs(n):
```

Comment:
This should probably defer to `validate_int_or_None` and then check that it's non-negative. And also should be private.